doc/policies/authentication.rst

```diff
@@ -419,6 +419,17 @@ there is no additional authentication step anymore during enrollment.
    enroll only one token type.
 
 
+.. _policy_enroll_via_multichallenge_text:
+
+enroll_via_multichallenge
```

Comment:
this should probably be `enroll_via_multichallenge_text`

Reply:
Fixed in 788c40c

Reply:
Thanks for the "allow_white_spaces".